community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -172,6 +172,10 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
     @Description( "Commands to be run when Neo4j Browser successfully connects to this server. Separate multiple commands with semi-colon." )
     public static final Setting<String> browser_postConnectCmd = setting( "browser.post_connect_cmd", STRING, "" );
 
+    @SuppressWarnings("unused") // accessed from the browser
+    @Description( "Whitelist of hosts for the Neo4j Browser to be allowed to fetch content from." )
+    public static final Setting<String> browser_remoteContentHostnameWhitelist = setting( "browser.remote_content_hostname_whitelist", STRING, "http://guides.neo4j.com,https://guides.neo4j.com,http://localhost,https://localhost" );
```

Comment:
It's enough to set the hostnames nowadays, so this could be changed to `guides.neo4j.com,localhost`.